packages/manager/apps/ips/src/pages/onboarding/onboardingIpOptionRow.tsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Links } from '@ovh-ux/manager-react-components';
+import { PriceDescription } from '@/components/PriceDescription/PriceDescription';
+import { useCatalogLowestPrice } from '@/data/hooks/catalog';
+import useGuideUtils from '@/pages/onboarding/useGuideUtils';
+
+interface IpOptionRowProps {
+  feature: string;
+  ipv4: React.ReactNode;
+  ipv6: React.ReactNode;
+}
+interface BulletPointListProps {
+  content: string;
+  feature: string;
+}
+
+const BulletPointList = ({ content, feature }: BulletPointListProps) => {
+  return content
+    ?.split('\n')
+    .map((line: string) => <div key={`${feature}-${line}`}>{`• ${line}`}</div>);
+};
+
+export const getIpOptionsRows = () => {
```

Comment:
```suggestion
export const useIpOptionsRows = () => {
```

Actually it's a hook since you use hooks inside it

And I think you can just do a component `IpOptionsRows` by returning the rendered rows directly since the hook will not be reused so no need to create a hook and export it.